src/open-hackathon-server/Kaiyuanshe.OpenHackathon.ServerTests/Controllers/UserControllerTests.cs

```diff
@@ -73,5 +73,32 @@ public async Task GetUserById()
         }
 
         #endregion
+
+        #region GetUploadUrl
+        [Test]
+        public async Task GetUploadUrl()
+        {
+            // input
+            var parameter = new FileUpload { expiration = 10, filename = "user/avatar.jpg" };
+            string url = "https://container.blob.core.chinacloudapi.cn/kysuser/user/avatar.jpg?sv=2018-03-28&sr=b&sig=JT1H%2FcjWK1xbwK3gy1qXXbXAxuQBTmYoCwpmK2Z9Ls0%3D&st=2021-07-04T09%3A25%3A39Z&se=2021-07-04T09%3A32%3A39Z&sp=rw";
+
+            // mock
+            var userManagement = new Mock<IUserManagement>();
+            userManagement.Setup(u => u.GetUploadUrl(parameter.expiration.GetValueOrDefault(), parameter.filename)).Returns(url);

```

Comment:
`userManagement.Setup(u => u.GetUploadUrl( 10,  "/user/avatar")).Returns(url);`
The second parameter should be `{currentUserid}/filename`.

Reply:
thanks. this is now fixed.